src/data-workspace/validation/validation-results-sidebar.module.css

```
@@ -0,0 +1,38 @@
+.wrapper {
+    margin: 16px
+}
+
+.butttons :global(button) {
+    margin-inline-end: 4px
+}
+
+.title {
+    color: var(--colors-grey900);
+    font-size: 16px;
+    line-height: 22px;
+    font-weight: 500;
```

Noticed this in the design spec 🙂

```suggestion
    font-weight: 500;
    margin-block-start: 8px;
    margin-block-end: 8px;
```